Bottom: 142e47d5243a177795955b2a72fb7ae5ea396377
Top:    3633af3bf18ea665c07ace5d795da5b07100235a
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-03-26 16:13:04 +0000

WAS: Use motion hints when processing pointer motion over the drawing area

Not sure what the remaining changes here are for!


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 46a7380..66bfa74 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -761,7 +761,7 @@ ghid_show_crosshair (gboolean paint_new_location)
 
   glBegin (GL_LINES);
 
-#if 1
+#if 0
   if (x_prev >= 0 && !paint_new_location)
     draw_crosshair (x_prev, y_prev);
 #endif
@@ -774,7 +774,7 @@ ghid_show_crosshair (gboolean paint_new_location)
 
   glBegin (GL_QUADS);
 
-#if 1
+#if 0
   if (x_prev >= 0 && draw_markers_prev & !paint_new_location)
     {
       glVertex2i (0,                  y_prev - VCD);
